edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -554,8 +671,10 @@ Scale2DQueryPageController = Class.create(QueryPageController, {
 
     this._min2 = this._radioListController2.getMin();
     this._max2 = this._radioListController2.getMax();
-    this._value2 = this._radioListController2.getSelected();  
+    this._value2 = this._radioListController2.getSelected();
+    this._updateNextButton();
   },
+  
   _initializeGraph: function (questionContainer) {
     this._graphController = new QueryBlockScaleGraphFragmentController(questionContainer);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Move the declaration of "QueryBlockScaleGraphFragmentController" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)